src/helpers/objects.processor.js

```diff
@@ -64,7 +64,7 @@ function pageObjectGetter (str) {
 function getPageObject (str) {
     const pageObjectGetterFunc = stepsConfig.objectsProcessor.pageObjectGetter || pageObjectGetter;
     const value = pageObjectGetterFunc(str);
-    const idValue = value.replace(_r(regDynamicId, 'g'), stepsConfig.id.getId());
+    const idValue = value.replace(_r(regDynamicId, 'g'), stepsConfig.id.id);
```

Comment:
`stepsConfig.id.id` - sounds a bit strange - maybe it will be better to name it `stepsConfig.id.idValue`?